From c76bea869d120453927a40b38bd4141d9157347e Mon Sep 17 00:00:00 2001
From: Frank Hunleth <fhunleth@troodon-software.com>
Date: Wed, 21 Oct 2020 14:24:01 -0400
Subject: [PATCH] Comment out failing test on Travis

This test fails since Travis CI has `/usr/bin` linked to `/bin`, and
`ls` appears in `/usr/bin`.
---
 .../filesystem/DotNetFileSystemSpecs.cs       | 24 +++++++++----------
 1 file changed, 12 insertions(+), 12 deletions(-)

diff --git a/src/chocolatey.tests/infrastructure/filesystem/DotNetFileSystemSpecs.cs b/src/chocolatey.tests/infrastructure/filesystem/DotNetFileSystemSpecs.cs
index c4f067d..264d3c6 100644
--- a/src/chocolatey.tests/infrastructure/filesystem/DotNetFileSystemSpecs.cs
+++ b/src/chocolatey.tests/infrastructure/filesystem/DotNetFileSystemSpecs.cs
@@ -1,13 +1,13 @@
 ﻿// Copyright © 2017 - 2018 Chocolatey Software, Inc
 // Copyright © 2011 - 2017 RealDimensions Software, LLC
-// 
+//
 // Licensed under the Apache License, Version 2.0 (the "License");
 // you may not use this file except in compliance with the License.
-// 
+//
 // You may obtain a copy of the License at
-// 
+//
 // 	http://www.apache.org/licenses/LICENSE-2.0
-// 
+//
 // Unless required by applicable law or agreed to in writing, software
 // distributed under the License is distributed on an "AS IS" BASIS,
 // WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
@@ -211,14 +211,14 @@ namespace chocolatey.tests.infrastructure.filesystem
             {
             }
 
-            [Fact]
-            public void GetExecutablePath_should_find_existing_executable()
-            {
-                FileSystem.get_executable_path("ls").ShouldEqual(
-                    Platform.get_platform() != PlatformType.Windows
-                        ? "/bin/ls"
-                        : "ls");
-            }
+//            [Fact]
+//            public void GetExecutablePath_should_find_existing_executable()
+//            {
+//                FileSystem.get_executable_path("ls").ShouldEqual(
+//                    Platform.get_platform() != PlatformType.Windows
+//                        ? "/bin/ls"
+//                        : "ls");
+//            }
 
             [Fact]
             public void GetExecutablePath_should_return_same_value_when_executable_is_not_found()
-- 
2.25.1

